vpd-tool/include/vpd_tool.hpp

```diff
@@ -17,6 +19,27 @@ namespace vpd
  */
 class VpdTool
 {
+  private:
+    /**
+     * @brief Get specific properties of a FRU in JSON format.
+     *
+     * For a given Object Path of a FRU, this API returns the following
+     * properties of the FRU in JSON format:
+     * - Present property, Pretty Name, Location Code, Sub Model
+     * - SN, PN, CC, FN, DR keywords under VINI record.
+     * - type and TYPE of the FRU
+     *
+     * @param[in] i_fruPath - DBus object path
+     *
+     * @return On success, returns the properties of the FRU in JSON format,
+     * otherwise returns an empty JSON object.
+     * Note: The caller of this API should
+     * check for empty JSON object
+     *
+     * @throw std::runtime_error
+     */
+    nlohmann::json getFruPropertiesJson(const std::string& i_fruPath) const;
```

Comment:
name doesn't gives the clarity on what this API is.
can you change it to something like - getFruProperties() -> as return type tells the format, it's not required to mention JSON format in API name.

Reply:
Done